src/components/Output/custom-els/TwoUp/index.ts

```diff
@@ -77,7 +89,8 @@ export default class TwoUp extends HTMLElement {
     // Set the initial position of the handle.
     requestAnimationFrame(() => {
       const bounds = this.getBoundingClientRect();
-      this._position = (this.orientation === 'vertical' ? bounds.height : bounds.width) / 2;
+      this._position = (this.orientation === 'vertical' ?
+          bounds.height : bounds.width) * this._relativePosition;
```

Comment:
Really minor minor nit: The wrapping of `(` and `)` is making me twitch. Can you switch it for something like:

```ts
this._position = (this.orientation === 'vertical' ? bounds.height : bounds.width)
  * this._relativePosition;
```

Or maybe duplicate the pattern below:

```ts
const dimensionAxis = this.orientation === 'vertical' ? 'height' : 'width';
this._position = bounds[dimensionAxis] * this._relativePosition;
```